test/Microsoft.ML.CpuMath.PerformanceTests/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.InProcess;
+
+namespace Microsoft.ML.CpuMath.PerformanceTests
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            BenchmarkSwitcher
+                .FromAssembly(typeof(Program).Assembly)
+                .Run(null, CreateClrVsCoreConfig());
+        }
+
+        private static IConfig CreateClrVsCoreConfig()
```

Comment:
Why is this needed?  Why can't we just use the default `static void Main(string[] args) => BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);`?

Reply:
When I removed the custom config function, the program tried to look for a `BenchmarkDotNet.Autogenerated` folder which did not exist, while having a custom config function has solved the error.  Will try the default `.Run(args);` tomorrow.

Reply:
Thanks for the suggestion!  I have tried `.Run(args)`, and it runs well, but if the user does not type any argument, the program returns swiftly and prints something like `Global Running Time: 0.00000010` on the standard output.  I originally wrote the function `CreateClrVsCoreConfig()` in order to comply with the existing code in `Program.cs` of `test\Microsoft.ML.Benchmark`:

https://github.com/dotnet/machinelearning/blob/8087a20f3294671b522d265ccfa07543c5ff6fa5/test/Microsoft.ML.Benchmarks/Program.cs#L29-L44

Please let me know how I should proceed - Thanks!

Reply:
No worries. We can leave it like this for now.